### ISSUE TITLE
io.jl test failure on Windows

### ISSUE THREAD

**Issue by tkelman** [timholy/Images.jl#324]
Continuing debugging from #316 #323 

```
julia> using Images, Compat

julia> Images.LibMagick.libwand
"C:\\Users\\Tony\\.julia\\Images\\deps\\usr\\lib\\x64\\CORE_RL_wand_.DLL"

julia> wand = Images.LibMagick.MagickWand()
MagickWand(Ptr{Void} @0x000000000017ad10)

julia> workdir = joinpath(tempdir(), "Images")
"C:\\Users\\Tony\\AppData\\Local\\Temp\\Images"

julia>  stream = open(joinpath(workdir, "2by2.png"))
IOStream(<file C:\Users\Tony\AppData\Local\Temp\Images\2by2.png>)

julia> Compat.Libc.FILE(stream).ptr
Ptr{Void} @0x000007feff2c2b10

julia> status = ccall((:MagickReadImageFile, Images.LibMagick.libwand), Cint, (P
tr{Void}, Ptr{Void}), wand.ptr, Compat.Libc.FILE(stream).ptr)
```

And that just crashes Julia. This functionality looks like a relatively recent addition from 3b0d46eb8777c067e08f76c0756dfd3714b22a16. I don't think it's valid to pass FILE objects across shared library boundaries on Windows when the shared libraries were built against different C runtimes - the ImageMagick that we're downloading is built against a recent MSVC runtime (msvcr120.dll), Julia is built against mingw and the old msvcrt.dll that's always present.

Switching to ImageMagick from WinRPM may fix this, because Julia and libwand would be using the same C runtime. I still haven't gotten a very clear story about what happened the last time that was tried, if it ever was. Another option would be to disable that test on windows.

**Comment by tkelman** [comment]
Other than the reading from a stream issue, it looks like there's only one other failure on 0.3 and its output is pretty long - https://gist.github.com/77a7cfe57e3e8b4ab222

**Comment by timholy** [comment]
Let's disable that test on Windows.

**Comment by tkelman** [comment]
Done in 1e295f356000431b1a2d9a394abe94139528fbae (included in #325) - that test was already being skipped on osx, the discrepancies are for 805 out of 11468 pixels.
